src/modules/organization/InternalGroupDetail.tsx

```diff
@@ -0,0 +1,180 @@
+import { useQuery } from '@apollo/client'
+import { UserThumbnail } from 'modules/users/UserThumbnail'
+import { useParams } from 'react-router-dom'
+import styled from 'styled-components'
+import { INTERNAL_GROUP_QUERY } from './queries'
+import { InternalGroupReturns, InternalGroupVariables } from './types'
+
+const Wrapper = styled.div`
+  ${props => props.theme.layout.default};
+  border-radius: 6px;
+  overflow-y: scroll;
+  display: grid;
+  grid-template-areas:
+    'groupname groupname . .'
+    'image image details details'
+    'members . . .';
+  ${props => props.theme.media.mobile} {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: space-between;
+  }
+  ${props => props.theme.media.largeScreen} {
+    grid-template-areas:
+      'groupname'
+      'image'
+      'details'
+      'members';
+  }
+  gap: 20px;
+  grid-template-columns: 1fr 2fr 2fr 2fr;
+
+  grid-template-rows: 72px 2fr 1fr;
+  width: 100%;
+  height: 100%;
+`
+const DetailsWrapper = styled.div`
+  grid-area: details;
+  display: grid;
+  grid-auto-rows: max-content;
+  background-color: white;
+  border-radius: 10px;
+  padding: 1.618em;
+  max-height: fit-content;
+`
+
+const DetailsHeader = styled.h3`
+  height: fit-content;
+  padding: 0 0 5px 0;
+  font-size: 1.7em;
+  border-bottom: solid rgba(100, 100, 100, 0.2);
+  border-width: 1px;
+`
+
+const DetailsText = styled.div``
```

Comment:
Should probably use styled.p (a styled HTML paragraph element) which is more semantically correct HTML in this case